b2/raw_api.py

```diff
@@ -454,11 +454,13 @@ def check_b2_filename(self, filename):
 
         :param filename: A proposed filename in utf-8.
         :return: None if the filename is usable."""
-        if len(filename) < 1:
+        decoded_filename = filename.decode("utf-8", 'ignore')
```

Comment:
Because the file name is already unicode, when I run this in Python 3, this line gets an error:

```
  File "/Users/brianb/sandbox/B2_Command_Line_Tool/b2/raw_api.py", line 457, in check_b2_filename
    decoded_filename = filename.decode("utf-8", 'ignore')
AttributeError: 'str' object has no attribute 'decode'
```

Reply:
Also fixed, since we are starting with unicode and utf-8 encoding it to get bytes.